### ISSUE TITLE
android_x86 support

### ISSUE THREAD

**Issue by liotcheg** [steeve/plugin.video.pulsar#158]
Thank you for this great addon!
Any chance for android_x86 support?
Recently a lot of such TV boxes appeared, and if you could compile a binary for android_x86 (and maybe android_x64), you would make all us (intel android tv boxes owners) really happy :) 

09:43:11 T:1910112304  NOTICE: OSError: [Errno 2] No such file or directory: '/storage/emulated/legacy/Android/data/org.xbmc.kodi/files/.kodi/addons/plugin.video.pulsar/resources/bin/android_x86'

**Comment by Acravetz** [comment]
I agree with lotcheg!!! Great app!!! I have a Minix Z64 android and can't install the app at all. Would love it to work. Waiting for updates!!!!

**Comment by Acravetz** [comment]
So nothing? No chance of making Pulsar available for Minix Z64 android?

**Comment by PLAY911** [comment]
Hi, please would be great if you give support to Android x86
Thank you

**Comment by lightglitch** [comment]
@steeve any news on this? It's a just compiling the golang app. 
If this part was open source I would do it and test it out.

**Comment by liotcheg** [comment]
This is a golang problem. It doesn't support android_x86 yet. Pulsar uses torrent2http, which is written in golang.
https://github.com/golang/go/issues/10743

**Comment by lightglitch** [comment]
Thx for the info. If I change the plugin to use the golang service on a
remote machine (the NAS) it might work.
